py/torch_migraphx/fx/converters/acc_ops_converters.py

```diff
@@ -325,6 +325,16 @@ def acc_ops_abs(mgx_module, node, args, kwargs):
                           qparams=inp.qparams)
 
 
+@migraphx_converter(acc_ops.logical_not)
+def acc_ops_logical_not(mgx_module, node, args, kwargs):
+    inp = kwargs["input"]
+    inp_bool = mgx_module.add_instruction(
+        migraphx.op('convert',  target_type=migraphx.shape.type_t.bool_type),
+        [inp.instr_ref])
+    return MGXInstruction(mgx_module.add_instruction(migraphx.op('not'),
+                                                     [inp_bool]))
```

Comment:
Dont need to add an explicit convert here, there is a parameter MGXInstruction takes for this case:
```suggestion
    return MGXInstruction(mgx_module.add_instruction(migraphx.op('not'),
                                                     [inp.instr_ref]), bool_output=True)
```

Reply:
nvm, this is doing it for the input. Thats probably necessary. Ignore the suggestion. You might still have to pass in the `bool_output` parameter. I would be surprised if the tests pass without it considering migraphx programs dont return booleans

Reply:
No, you're right the first time.  changed.